package.json

```diff
@@ -1,8 +1,28 @@
 {
-  "name": "pslint",
+  "name": "eslint-config-primarysite",
   "version": "0.0.0",
```

Comment:
Even though we're not actually publishing this to npm yet (or ever), should we still give this a version? Will we be creating and tagging releases of this repo, then pulling them in via npm using the tag? * curious *